### PR TITLE
198 discount code parser docs

### LINKED ISSUE #198: [csv-parser-discount-code] Add documentation
### Description

Add documentation for the csv-parser-discount-code module (#197)

### PR DESCRIPTION
**Pull request by wizzy25**
#### Summary

Add documentation for the [csv-parser-discount-code] #197

resolves #198

### CODE REVIEW

**Comment by wizzy25** [comment]
Failing build will pass when #197 gets merged

File: packages/csv-parser-discount-code/src/main.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function (a, b) {
+  return a * b
```
Comment (adnasa) [inline packages/csv-parser-discount-code/src/main.js]: wait... this is it? 💃 😆

Comment (hisabimbola) [inline packages/csv-parser-discount-code/src/main.js]: hehehe, pseudocode

Comment (wizzy25) [inline packages/csv-parser-discount-code/src/main.js]: 😅😅😅

**Pull request merged**